src/alarm_trap_sender.cpp

```diff
@@ -143,3 +166,112 @@ void AlarmTrapSender::send_trap(const AlarmTableDef& alarm_table_def)
 
   snmp_reset_var_buffers(&var_trap);
 }
+
+void AlarmTrapSender::send_enterprise_trap(const AlarmTableDef& alarm_table_def)
+{
+  static const unsigned int severity_to_state[] = {2, 1, 2, 6, 5, 4, 3};
+  static const oid snmp_trap_oid[] = {1,3,6,1,6,3,1,1,4,1,0};
+  static const oid trap_type_oid[] = {1,3,6,1,4,1,19444,12,2,0,1};
+  static const oid MIB_version_oid[] = {1,3,6,1,4,1,19444,12,1};
```

Comment:
This isn't the OID of clearwaterInfoVersion according to the MIB (it's .1.2.826.0.1.1578918.12.1.1 there) - not sure whether this is wrong or the MIB file is wrong.